packages/playwright/src/common/config.ts

```diff
@@ -96,6 +97,11 @@ export class FullConfigInternal {
       workers: 0,
       webServer: null,
     };
+    for (const key in config) {
+      if (key.startsWith('@'))
+        (this.config as any)[key] = (config as any)[key];
```

Comment:
I wonder whether we `JSON.stringify()` the config somewhere, which would probably break?

Reply:
I hope we never do that